### DISCUSSION TITLE
undefined referance to function hatası

### DISCUSSION THREAD

**Discussion by ghost** [19/classroom#61] [Anonim]
math.h kitaplığını kullanmama rağmen undefined referance to log hatası veriyor. Lütfen yardımcı olur musunuz?

**Comment by roktas** [comment]
Derlerken `-lm` anahtarını ekleyeceksin.

```sh
cc -o main -lm main.c
```

**Reply by ghost** [reply]
Bunu github üzerinde nasıl yapabilirim?

**Comment by roktas** [comment]
Yazdığım açıklama genel idi (kendi makinende yapılan derlemeler için).  Bu ödevde `math.h` kullanmanı gerektiren bir şey yok, o nedenle testlerde `-lm` anahtarı kullanmadık.  Çözüm yöntemini değiştir, matematiksel bir fonksiyon kullanman gerekmiyor ödevde.

**Reply by ghost** [reply]
Tamamdır hocam teşekkürler
